helm-charts/splunk-otel-collector/templates/operator/_helpers.tpl

```diff
@@ -153,7 +153,7 @@ Helper for generating environment variables for each instrumentation library.
 
   {{- /* Handle custom or default exporter endpoint */ -}}
   {{- $customOtelExporterEndpoint := "" }}
-  {{- if or (eq .instLibName "dotnet") (eq .instLibName "python") (eq .instLibName "java") }}
+  {{- if or (eq .instLibName "dotnet") (eq .instLibName "java") (eq .instLibName "nodejs") (eq .instLibName "python") }}
     {{- $customOtelExporterEndpoint = .endpoint | replace ":4317" ":4318" }}
```

Comment:
We should consider if we still have cases that need this conditional. In principle we could just default to 4318.